src/CL/Slack/Model/Attachment.php

```diff
@@ -257,4 +262,20 @@ public function getFields()
     {
         return $this->fields;
     }
+    
+    /**
+     * @param array Valid values for mrkdwn_in are: ["pretext", "text", "fields"]. Setting "fields" will enable markup formatting for the value of each field
+     */
+    public function setMrkdwnIn(Array $mrkdwnIn)
```

Comment:
Although the API may use this naming (`mrkdown_in`), I prefer having our library use clear method names, could you refactor the setters and getters to `setMarkdownIn` and `getMarkdownIn` respectively? You can leave the property name as is, because that will be needed for the actual communication with Slack (and therefore must follow their schema).

Oh and can you also fix the type hint here from `Array` to `array`, thanks!